consensus/byzantine_test.go

```diff
@@ -86,6 +79,16 @@ func TestByzantine(t *testing.T) {
 		reactors[i] = conRI
 	}
 
+	defer func() {
```

Comment:
Why did we move this down here?

Reply:
it just felt more natural because the order is always: `file.Open(); defer f.Close()` and not the other way around